torch/distributed/fsdp/_optim_utils.py

```diff
@@ -13,24 +12,19 @@
     Sequence,
     Tuple,
     Union,
+    cast,
```

Comment:
nit: looks like the sorting policy here (ASCII) is different from the one used in `test/distributed/fsdp/test_fsdp_exec_order.py` (alphabet).

Reply:
I think ufmt uses a different sorting policy than the VSCode sort imports. I need to look into this.